cmd/agent/app/options/options.go

```diff
@@ -80,6 +81,7 @@ func (o *GrpcProxyAgentOptions) Flags() *pflag.FlagSet {
 	flags.StringVar(&o.ProxyServerHost, "proxy-server-host", o.ProxyServerHost, "The hostname to use to connect to the proxy-server.")
 	flags.IntVar(&o.ProxyServerPort, "proxy-server-port", o.ProxyServerPort, "The port the proxy server is listening on.")
 	flags.StringSliceVar(&o.AlpnProtos, "alpn-proto", o.AlpnProtos, "Additional ALPN protocols to be presented when connecting to the server. Useful to distinguish between network proxy and apiserver connections that share the same destination address.")
+	flags.StringVar(&o.HealthServerHost, "health-server-host", o.HealthServerHost, "The host address to listen on, without port.")
```

Comment:
I think the general naming convention for this is:
```
--health-server-addr
```

Reply:
Ah, I see this comment abbove now https://github.com/kubernetes-sigs/apiserver-network-proxy/pull/313#discussion_r772608611

Reply:
> Ah, I see this comment abbove now #313 (comment)

sorry for the confusion, ^ when I said this I meant you can ignore my comment :)

Reply:
(I think it's better to be consistent with --proxy-server-host, and while it'd be nice if both ended in `-addr`, I don't think it's worth breaking the existing flag)

Reply:
ok, changed the name back to `health-server-host`